views/markdown.cfm

```diff
@@ -1,11 +1,11 @@
 <cfif structKeyExists(variables, "data")>
 	<cfif find("<h1>", variables.data)>
 		<cfset request.title = reReplace(variables.data, ".*<h1>([^<]+)<.+", "\1", "ONE")>
+		<cfset request.title &= "<small><a href="https://github.com/foundeo/cfdocs#request.gitFilePath#" rel="nofollow"><span class="glyphicon glyphicon-pencil"></span></a></small>">
```

Comment:
I don't think this will put the edit button on the page, it will put it in the `<title>` tag I think, right?